engines/e_devcrypto.c

```diff
@@ -73,12 +83,12 @@ struct driver_info_st {
 void engine_load_devcrypto_int(void);
 #endif
 
-static int clean_devcrypto_session(struct session_op *sess) {
+static int clean_devcrypto_session(session_op_t *sess) {
     if (ioctl(cfd, CIOCFSESSION, &sess->ses) < 0) {
         ERR_raise_data(ERR_LIB_SYS, errno, "calling ioctl()");
         return 0;
     }
-    memset(sess, 0, sizeof(struct session_op));
+    memset(sess, 0, sizeof(*sess));
```

Comment:
While this could have just used the typedef, my understanding is that using the size of the variable is generally preferred.  It is also more consistent with other uses of `memset` in this file which use the size of the variable rather than the size of the type.

Reply:
yes, this is a good code cleanup thing to do, in case the type of the variable changes.  +1!